ezyfox-server-core/src/main/java/com/tvd12/ezyfoxserver/socket/EzySocketWriter.java

```diff
@@ -16,27 +16,26 @@ public class EzySocketWriter
 
     @Override
     public void handleEvent() {
-        doProcessSessionTicketsQueue();
-    }
-
-    @Override
-    public void destroy() {
-        processWithLogException(() -> sessionTicketsQueue.clear());
-    }
-
-    private void doProcessSessionTicketsQueue() {
         try {
             EzySession session = sessionTicketsQueue.take();
             processSessionQueue(session);
         } catch (InterruptedException e) {
             logger.info("socket-writer thread interrupted");
-        } catch (Throwable throwable) {
-            logger.warn("problems in socket-writer, thread", throwable);
+        } catch (Throwable e) {
+            logger.info("problems in socket-writer, thread", e);
```

Comment:
I'm not sure how to set log level here? It looks like we should use `logger.error`, but I could be wrong

Reply:
Yes, because `warn` or `error` make spam logs on the monitoring channel in some normal case (example, client disconnected from server)